Thesis.tex

```diff
@@ -169,6 +169,20 @@ \subsection{Closed Source Software}
+
+These limitations were also applied to works produced in America. Interestingly enough, even works which were considered protected later on, were not eligible for copyright protection, if they were found to be of illicit nature.  Exactly this happened to the work \textit{Three Weeks} by Elinor Glynn, which was deemed so \textquotedblleft grossly immoral that copyright protection cannot exist\textquotedblright\ as is described in \cite{Fox1945}.
+We can conclude that piracy was if not necessarily as widespread as it is today, it was prevalent enough, that voices for judicial protection never ceased to exist.
+
+The same was true for the creators of music in the 1900s, as sheet music was not deemed \textquoteleft useful arts\textquoteright\ and therefore not worthy of protection. Even after sheet music was being protected by the 1909 copyright act, which introduced the ingenious construct of 'compulsory licensing' (in which the creator can choose who produced the first recording of his music, and every other recorder just has to pay a flat fee), the recordings of said music was not itself protected. The justification of this disparity was that these recordings were merely parts of a machine and thus not a creation protected by copyright.
+
+This led to a proliferation of copied LPs. Especially bootlegging rare Jazz albums was common in the 1930, and even tolerated by the major labels as one executive told: \textquotedblleft Ah, it wasn't worth the trouble to put out that moldy stuff [...] It never sold anyway\textquotedblright\ \parencite{Cummings2017}
+In the 1960 the flourishing of pirated music on magnetic tape and vinyls prompted the music industry to push for another try at protecting their products. This time around their narrative was not some ideal goal of authorship, but their own bottom line and how piracy threatened the American industry and jobs. This push was more successful than the preceding ones and lead to records being protected under American law.
+
+In the same time, the efflorescence of radio and television led to the development of, what \cite{Lin2013} calls, the mentality of free. The customer is receiving content for \textquotedblleft free\textquotedblright\ from service providers, which in turn pay the licensing fee and use advertising to pay these and profit from their service. This business model was broadly adopted in the age of internet. The problem being, that the customer had to listen or view advertisements on radio or television, banner ads could be more easily ignored. During this time Peer-to-peer filesharing systems like Napster started to spring up, propelled by the ever increasing speeds of the domestic internet connection. And quoting from the excellent paper from \cite{Lunceford2008}: \textquotedblleft The logic of the consumer dictates that if a product is freely available on the market [through service providers], it is probably legal.\textquotedblright
```

Comment:
The problem being that the customer had to listen or view advertisements on radio or television; banner ads, however, could be more easily ignored.